lib/main.js

```diff
@@ -322,7 +323,8 @@ module.exports = class Router extends EventEmitter {
     }
 
     statusErrors() {
-        return (error, req, res, next) => { // eslint-disable-line
+        // eslint-disable-next-line
+    return (error, req, res, next) => {
```

Comment:
grrrrr

Reply:
We talked about it, but just make the ignore more specific, and eslint should fix it